src/rdf4cpp/storage/reference_node_storage/detail/BiDirFlatMap.hpp

```diff
@@ -97,7 +97,11 @@ struct BiDirFlatMap {
      * Translates the given id into an index into forward_
      */
     [[nodiscard]] static constexpr size_type to_index(id_type const id) noexcept {
-        return static_cast<size_type>(id) - 1;
+        if constexpr (std::same_as<uint64_t, size_t>) {
```

Comment:
I think it would probably be better if we just used `uint64_t` instead of `size_t` for the bitmap type